src/scripts/generate_relation_and_equality_ops.py

```diff
@@ -0,0 +1,271 @@
+            return GenSequence.create_operation_result_object(self ,'>=', right_value, "false")
+        result = "true" if self.less(left_value, right_value)['result'] == "false" else "false"
+        return GenSequence.create_operation_result_object(self, '>=', right_value, result)
+
+    # Declaration creater function
+    def create_declarations(self, expected_value):
+        # Adding false boolean string
+        false_boolean_string = self.create_false_boolean_string(expected_value)
+        # Adding commas if value is meant to be a string
+        for i in range(len(self.operands)):
+            self.operands[i] = self.add_coma_if_string(self.operands[i])
+        # Adding declarations of test values
+        test_case = self.create_declarations_string(self.create_declarations_dict(self.operands,
+        self.TEST_VALUE_VAR_NAME))
+        # Adding declarations of false value
+        test_case += self.create_declarations_string(self.create_declarations_dict(false_boolean_string,
+        self.FALSE_RESULT_VAR_NAME))
+        # Adding declarations of expected value
+        test_case += self.create_declarations_string(self.create_declarations_dict(expected_value,
+        self.EXPECTED_VAR_NAME))
+        return test_case
+
+    # Converting expression, expected value, false value in a function call to string format
+    def string_creator(self):
+        self.expression_generator()
+        last = self.operands[0]
+        expression = "%s%s" % ("(" * (self.num_operands - 1), self.TEST_VALUE_VAR_NAME.format(NUMBER = (1)))
+        for i in range(self.options.operand_count - 1):
+            result = self.operators_list[self.operators[i]](last, self.operands[i + 1])
+            expression += (" %s %s)" % (result["oper"], self.TEST_VALUE_VAR_NAME.format(NUMBER = (i + 2))))
+            self.operands[i + 1] = result["right_value"]
+            last = result["result"]
+        expression += "\n"
+        test_case = self.create_declarations(last)
+        # Adding the function call
+        test_case += self.create_function_call(expression, self.EXPECTED_VAR_NAME.format(NUMBER = ""),
+        self.FALSE_RESULT_VAR_NAME.format(NUMBER = ""), validate.validate_boolean_header)
+        test_case += "\n\n"
+        return test_case
+
+    # Relational and equality operator test generator function
+    def generate_relational_and_equality_ops(self, options):
+        random.seed(self.options.seed)
+        self.debug(Messages.generating, self.options)
+        gen_settings = settings.file_write_settings(self._InnerValues.generated_filename, self.options.output, validate.validate_boolean,
+        _constants.test_case_in_a_file)
+        for i in range(self.options.test_count): # In range of number test cases
+            self.test_case += self.string_creator()
+            self.append_test_case()
+        write_file(gen_settings, self.file_output)
+        self.debug(Messages.done, self.options)
+
+def main():
```

Comment:
Why do you create separate `main` function? Do you want to call this why you are importing this class?
If you do not want that the `main` will be callable, please move its body under `if __name__ == '__main__':` block!